functions/exercises.py

```diff
@@ -18,12 +17,16 @@ def get_max_of_three(x, y, z):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    if x>y and x>z:
+        return x
+    elif y > x and y>z:

```

Comment:
```suggestion
    if x > y and x > z:
        return x
    elif y > x and y > z:

```
nitpicking: making it visually more pleasant